add description of what the productHeader actually does (#568)
commerce-docs/microsite-commerce-storefront: src/content/docs/dropins/product-details/containers/product-header.mdx

The `ProductHeader` container is designed to display the header information of a product on the product details page.

> The header displays the product name with the SKU shown underneath. When the `hideSku` option is set to `true`, the SKU is not displayed, showing only the product name.

Reviewer: ```suggestion
The header displays the product name with the SKU below it. Set `hideSku` to `true` to hide the SKU.
```
